javascript/statements.json

```diff
@@ -331,27 +331,9 @@
                 ]
               }
             ],
-            "opera_android": [
-              {
-                "version_added": "36"
-              },
-              {
-                "version_removed": "36",
-                "version_added": "29",
-                "notes": "Strict mode is required."
-              },
```

Comment:
Dropping this looks suspicious or out of scope for this PR.

Reply:
Digging into this some more, I think I see how the data is supposed to be read.  In #6012, this was restructured from a note into multiple segments, however when I was reading through the data, I thought that the compat sections meant that both strict mode _and_ enabling the flags were required, not either-or.  Luckily though, this data falls under the scope of irrelevant flags, so I'll open a separate PR to take care of them for all of Chromium!

Reply:
Deferred to #15643.